src/utils/system.rs

```diff
@@ -217,5 +217,5 @@ pub fn get_model(config: &Config) -> Option<String> {
 
 #[cfg(not(target_os="macos"))]
 pub fn get_family(config: &Config) -> Option<String> {
-    config.get_family()
+    Some(config.family.to_owned())
```

Comment:
This fixes my earlier comment. But the fact that `config.get_family()` and `config.family` don't always return the same value probably means I'm doing this wrong.